### PR TITLE
fix lk_to_pk/3 wrt irreg keys, move it to riak_ql_ddl

### PR DESCRIPTION
**Pull request by hmmr**
Depends on https://github.com/basho/riak_ql/pull/158.

This unbreaks ts_cluster_comprehensive, ts_cluster_list_irreg_keys_SUITE and ts_cluster_stream_list_keys_SUITE.

@andytill to review.

### CODE REVIEW

**Comment by andytill** [comment]
This breaks the `ts_cluster_stream_list_keys_SUITE` suite.

File: src/riak_kv_ts_util.erl [hunk 1]
```diff
@@ -405,61 +402,12 @@ try_compile_ddl(DDL) ->
     {ok, _, _} = compile:forms(AST),
     ok.
 
-
--spec make_ts_keys([riak_pb_ts_codec:ldbvalue()], ?DDL{}, module()) ->
```
Comment (hazen) [inline src/riak_kv_ts_util.erl:409]: So this is no longer needed?

Comment (hmmr) [reply src/riak_kv_ts_util.erl:409]: Not needed, verified by grep.

**Comment by hazen** [comment]
@andytill I pulled fresh `riak` and `riak_test` and used @hmmr's changes, but `ts_cluster_stream_list_keys_SUITE` seems to pass for me:
```
Test Results:
ts_cluster_stream_list_keys_SUITE-riak_kv_bitcask_backend,riak_kv_eleveldb_backend: pass
---------------------------------------------
0 Tests Failed
1 Tests Passed
That's 100.0% for those keeping score
```

**Comment by hmmr** [comment]
@javajolt I have made changes since (and in response to) the first report of failing tests by @andytill. Also, make sure ts_cluster_list_irreg_keys passes (originally reported to fail by @korry8911).

**Comment by hazen** [comment]
```
ts_cluster_list_irreg_keys_SUITE-riak_kv_bitcask_backend,riak_kv_eleveldb_backend: pass
---------------------------------------------
0 Tests Failed
1 Tests Passed
That's 100.0% for those keeping score
```

**Comment by hazen** [comment]
FWIW - `ts_cluster_comprehensive` still fails

**Comment by hmmr** [comment]
@javajolt it fails for an unrelated reason (there are problems with API calls to nonexistent tables, something Andy or James are working to fix).

**Comment by andytill** [comment]
+1 a8c0bb2

**Comment by hazen** [comment]
@borshop merge

**Pull request merged**